utilities_common/cli.py

```diff
@@ -576,3 +576,32 @@ def interface_is_untagged_member(db, interface_name):
             if (val['tagging_mode'] == 'untagged'):
                 return True
     return False
+
+def is_interface_in_config_db(config_db, interface_name):
+    """ Check if an interface is in CONFIG DB """
+    if (not interface_name in config_db.get_keys('VLAN_INTERFACE') and
+        not interface_name in config_db.get_keys('INTERFACE') and
+        not interface_name in config_db.get_keys('PORTCHANNEL_INTERFACE') and
+        not interface_name == 'null'):
+            return False
+
+    return True
+
+def is_interface_in_appl_db(appl_db, interface_name):
+    """ Check if an interface is in APPL DB """
+    key = "INTF_TABLE:" + str(interface_name)
+    interfaces = appl_db.get_all(appl_db.APPL_DB, key)
+
+    if interfaces is None:
+        return False
+
+    return True
+
+def get_all_interfaces(appl_db):
```

Comment:
if there's no other logic let's just use "appl_db.keys(appl_db.APPL_DB, "INTF_TABLE:*")" This itself is an API.